### DISCUSSION TITLE
When swapping windows the backgrounds briefly bleeds through. Can I make swapping as instantly as possible?

### DISCUSSION THREAD

**Discussion by mahulst** [regolith-linux/regolith-desktop#932] [Q&A] [answered]
Hi, 
Quite new to regolith/i3 and search for a bunch of keywords (animation, instantly swapping, bleeding background, etc) but did not found what I was looking for:
I want to switch between windows without any animation, just instantly swap the next application.
Any pointers/help would be greatly appreciated.

**Comment by kgilmer** [comment, marked as answer]
I assume you're running the X11 session.  These instructions would not work for the Wayland session.  The program you want to configure or remove is the compositor.  Regolith offers three implementations, and two of them can be customized.  The simplest thing is to install `regolith-compositor-none` and then restart your session for maximum speed.  If you'd like some visual enhancements, you would then look into the configuration for compton or picom and set it up based on your desires.

```
~$ apt list | grep regolith-compositor-

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

regolith-compositor-compton-glx/unknown 1.2.0-1regolith-lunar amd64
regolith-compositor-none/unknown 1.0.3-1-1regolith-lunar amd64
regolith-compositor-picom-glx/unknown,now 1.3.0-1regolith-lunar amd64 [installed]
regolith-compositor-xcompmgr/unknown 1.3.0-1-1regolith-lunar amd64
```

https://regolith-desktop.com/docs/howtos/customize-compositor/

**Reply by mahulst** [reply]
Worked! Exactly what I was looking for, thanks a bunch.

**Reply by harold** [reply]
@kgilmer - you've been doing great work for a long time, just want to express my gratitude here again. I'm upgrading bits of regolith presently and everything is awesome.

Long live `-none` the compositor and its indelibly efficient operation. :bow:
